scripts/initscripts/snapd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
This service name conflicts with a package installed by default in Ubuntu 16.04.1, so we are going to change it. See: https://github.com/intelsdi-x/snap_packaging/issues/13